### ISSUE TITLE
Incorrect light effect saved to scene.yaml

### ISSUE THREAD

**Issue by Erriez** [home-assistant/frontend#8062]
## Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

## The problem

Issue 1: All effects listed in a light effect drop-down are saved in `scene.yaml` `effect_list:` instead of `effect:` for the selected effect.

Issue 2: Cannot deselect an affect in light Effect drop-down.

Issue 3: Changing an effect in the light effect drop-down does not show the `Save` button.

## Expected behavior

Issue 1: Remove `effect_list:` and add `effect:` for the selected light effect in `scene.yaml`:

Issue 2: There should be a possibility to deselect an effect in the drop-down to default.

Issue 3: Show the `Save` button after changing the effect drop-down.

## Steps to reproduce

![HomeAssistantBugSaveSceneEffect3](https://user-images.githubusercontent.com/8849873/103442155-f026c000-4c53-11eb-881b-0baff8e5cf11.gif)

## Environment

- Home Assistant release with the issue: Home Assistant OS: 2020.12.2; Home Assistant OS 5.9
- Last working Home Assistant release (if known): Unknown
- Browser and browser version: Latest Firefox / Chrome
- Operating system: Not applicable

## State of relevant entities

Not applicable.

## Problem-relevant configuration

All effects are saved to `effect_list:` as listed in light effect drop-down which is incorrect:

```yaml
- id: '1609517205223'
  name: New Scene
  entities:
    light.milight_fut089_0x55b1_zone_1:
      min_mireds: 153
      max_mireds: 500
      effect_list:  <- Remove
      - night_mode  <- Remove
      - white_mode  <- Remove
      brightness: 23
      color_temp: 229
      friendly_name: MiLight FUT089 0x55B1 Zone 1
      supported_features: 63
      state: 'on'
```

Correct working selected effect:

```yaml
- id: '1609517205223'
  name: New Scene
  entities:
    light.milight_fut089_0x55b1_zone_1:
      min_mireds: 153
      max_mireds: 500
      effect: white_mode  <- Or remove line when no effect selected
      brightness: 23
      color_temp: 229
      friendly_name: MiLight FUT089 0x55B1 Zone 1
      supported_features: 63
      state: 'on'
```

Note: This change is overwritten every time when opening/modifying a scene, so a manual modification is not a solution.

## Javascript errors shown in your browser console/inspector

Not applicable.

## Additional information

See screen capture above.

With the issues listed above, [MiLight-Hub](https://github.com/sidoh/esp8266_milight_hub) cannot switch between RGB and white mode as the effect is not saved correctly in a scene.

**Comment by Timofiev** [comment]
I discovered this a long time ago and finally I found almost the same solution myself.

When I created a scene in the Scene editor in HA and set up the brightness and color_temp to my preferences after saving and running the scene it activated a completely different setting than what I set up earlier.

My solution was to delete the hs_color, rgb_color and xy_color and all effect_list lines from scenes.yaml (basically everything not relevant). After this my scene worked as I wanted. I don't know why the Scene editor in HA puts these parameters into the scenes.yaml, but it don't let to set up the scenes correctly.
